cyberglove_trajectory/src/cyberglove_trajectory/cyberglove_mock.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2019 Shadow Robot Company Ltd - All Rights Reserved.
+# Proprietary and Confidential. Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import rospy
+import rospkg
+import argparse
+import actionlib
+import tty
+import sys
+import tf
+import tf2_ros
+import yaml
+import geometry_msgs.msg
+import termios
+from std_srvs.srv import Trigger, TriggerRequest
+from threading import Thread, Lock
+from sr_vive_tracking.msg import ControllerEvent
+from control_msgs.msg import FollowJointTrajectoryAction, FollowJointTrajectoryGoal
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+
+
+class VRCodes:
+    k_EButton_ApplicationMenu = 1
+    k_EButton_SteamVR_Trigger = 33
+    VREvent_ButtonPress = 200
+    VREvent_ButtonUnpress = 201
+    TrackedDeviceClass_Controller = 2
+
+class CybergloveMock(object):
+    def __init__(self, cycle_hand_positions=True):
+        self._cycle_hand_positions = cycle_hand_positions
+
+        self._button_publisher = rospy.Publisher('sr_vive_controller_events', ControllerEvent, queue_size=5)
+        self._hand_traj_client = actionlib.SimpleActionClient('/sr_teleop_vive_cyberglove_trajectory_controller' +
```

Comment:
this should really be `"rh_trajectory_controller/follow_joint_trajectory"` by default. Only when running it for teleop, we should remap that to this value, like is being done here https://github.com/shadow-robot/sr_teleop_demos/blob/kinetic-devel/sr_teleop_vive_cyberglove/launch/cyberglove.launch#L9

Guess you should just parametrize it and in the top level mock for teleop overwrite to proper value.

Reply:
Also, you will have to change that default here:
https://github.com/shadow-robot/sr_teleop_demos/pull/47/files#diff-548d6b3dbc683e6303cde66c0ff034f7R19